src/github_repo_request_local.py

```diff
@@ -0,0 +1,74 @@
+import subprocess
+from pathlib import Path
+import pandas as pd
+from loguru import logger
+import shutil
+
+
+def count_and_list_test_files(directory):
```

Comment:
How about making this function simpler by getting it to do one thing - provide a list of the test files? the count can be performed trivially on the list that's returned by the calling code (e.g. line 66 here).

Reply:
Soryy I missed this :)
- Changed the name of the function from `count_and_list_test_files(clone_dir)` to 
'list_test_files(clone_dir)'